core/commands/add.go

```diff
@@ -29,6 +31,7 @@ const progressReaderIncrement = 1024 * 256
 const (
 	progressOptionName = "progress"
 	wrapOptionName     = "wrap-with-directory"
+	hiddenOptionName   = "hidden"
```

Comment:
On Thu, May 07, 2015 at 12:01:54PM -0700, Jeromy Johnson wrote:
> > @@ -29,6 +31,7 @@ const progressReaderIncrement = 1024 * 256
> >  const (
> >  	progressOptionName = "progress"
> >  	wrapOptionName     = "wrap-with-directory"
> > +	hiddenOptionName   = "hidden"
> 
> i'm terrible at UX, but i expect dotfiles and hidden files to be
> ignored by default

For what it's worth, Git does not ignore dotfiles by default.  I'm not
sure how they handle Windows' hidden attribute.